com.pypyrev.linuxdcpp.json

```diff
@@ -0,0 +1,74 @@
+{
+	"app-id": "com.pypyrev.linuxdcpp",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "18.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "linuxdcpp",
+	"finish-args": [
+		"--socket=x11",
+		"--share=ipc",
+		"--share=network",
+		"--filesystem=host"
```

Comment:
do you really need host access?

Reply:
User may want to download to any directory or share any directory (like in original app). So, I think that it's better to provide this ability without additional setup.